SAPHana/SAPHanaSR-ScaleOut.spec

```diff
@@ -123,6 +125,8 @@ install -m 0444 srHook/sudoers %{buildroot}/usr/share/%{name}/samples
 /usr/sbin/SAPHanaSR-manageAttr
 /usr/sbin/SAPHanaSR-replay-archive
 /usr/sbin/SAPHanaSR-filter
+/usr/sbin/SAPHanaSR-SAPHanaSR-show-hadr-runtimes
+/usr/sbin/SAPHanaSR-SAPHanaSR-show-hadr-log-coincidence
```

Comment:
Sorry. I overlooked the wrong names in the first review
SAPHanaSR-SAPHanaSR-show-hadr-runtimes and SAPHanaSR-SAPHanaSR-show-hadr-log-coincidence
 have a 'SAPHanaSR-' simply too much.

Reply:
Now I know why the space came in. I will repair that ASAP. Sorry that I am bothering you with my typos. We need to work on the fact that I need to do my own test-builds, so I would find such typos before I send the PR. In the past I did build the packages in the IBS by manually using osc but I thing you have something better, which build after sending commits.

Reply:
No, sorry. No automation for SAPHanaSR and SAPHanaSR-ScaleOut at the moment. Simply building in IBS manually.